notebooks/featurestore/hsfs/feature_engineering.ipynb

```diff
@@ -310,28 +312,6 @@
     "When creating a feature group we can also specify a `partition key`. Partition keys help organize the feature data on the file system and improve perfomances when reading the feature group data. As for the `primary key`, also `partition key` can be a composite one."
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": 13,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "<hsfs.feature_group.FeatureGroup object at 0x7fd3e4d40850>"
-     ]
-    }
-   ],
-   "source": [
-    "sales_part_fg_meta = fs.create_feature_group(name=\"sales_fg\",\n",
-    "                                        version=2,\n",
-    "                                        partition_key=['store'],\n",
```

Comment:
Why did you remove the partition key example?

Reply:
My bad. Its back now